### ISSUE TITLE
Adding MapId property for map styles

### ISSUE THREAD

**Issue by vanessa-myt** [google-map-react/google-map-react#1157]
I haven't read anything about mapIds here in the repo. So please enlighten me incase this feature was already created.

I wanted to style my map using cloud-based styling on Google Maps Platform. Can we have a mapId property in order to apply the styles I created on GCP? Thanks!

**Comment by strarsis** [comment]
@vanessa-myt: Yes, it is already possible, though with a JavaScript warning (which doesn't cause issues):
https://github.com/google-map-react/google-map-react/issues/1070#issuecomment-1008249855
